modules/ROOT/pages/preview-community.adoc

```diff
@@ -0,0 +1,19 @@
+= Preview Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+To show a live preview of the community's site, including APIs in *Preview Mode*, open the ACM administrator interface and select *Community Builder*.
+
+image::builder.png[Site Builder]
+
+Select *Preview*.
+
+To return to the site builder, select *Back to Builder*.
+
+Use *Community Builder* to *Publish* the community.
```

Comment:
Make this a separate topic: a task, with an introduction providing context and user benefit, and steps.

Reply:
If you retain this text, do not use the UI labels to represent names, common nouns, or verbs:

Use Community Builder [or the Community Builder tool/component/...] to publish your finished community:

1. From the API Community Manager (ACM) interface, click *Community Workspace*.
2. Click *Active*.

You can view your published community from the ACM interface by clicking *Open Community*.

Reply:
This has been separated into two task topics for preview and publish, with intros providing context and user benefit, and steps.